token_auth/views/register.py

```diff
@@ -14,15 +21,43 @@ class RegisterView(APIView):
     permission_classes = [AllowAny]
 
     def post(self, request, *args, **kwargs):
-        
+        """
+        Function for creating user account
+        """
+
+        # Account Creation
         serializer = RegisterSerializer(data=request.data)
         if not serializer.is_valid():
             return Response(serializer.errors, status.HTTP_400_BAD_REQUEST)        
         user = serializer.save()
-        refresh = RefreshToken.for_user(user)
-        res = {
-            "refresh": str(refresh),
-            "access": str(refresh.access_token),
-        }
-        return Response(res, status.HTTP_201_CREATED)
+
+        # Send Email for confirmation
+        current_site = get_current_site(request)
+        send_mail(
+            'Confirmation Email',
+            f'{current_site.domain}: {urlsafe_base64_encode(force_bytes(user.pk))}: {account_activation_token.make_token(user)}',
+            'bg566528@gmail.com',
```

Comment:
This doesn't necessarily have to be a valid email, since we are using sendgrid's API key.
`<OSP Admin> osp@anitb.org` would also work. Or contact @mayburgos for what should be the sender name and email